packages/ethereum-contracts/package.json

```diff
@@ -21,11 +21,11 @@
         "/build/abi.js",
         "artifacts/contracts/**/*.json",
         "!/**/*.dbg.json",
-        "scripts/**/*",
+        "dev-scripts/**/*",
```

Comment:
let's leave "ops-scripts" there still.

also if we add compatibility "scripts" placeholders, add her too.